x-pack/plugins/apm/public/components/app/Main/__test__/__snapshots__/Home.test.js.snap

```diff
@@ -31,9 +31,7 @@ exports[`Home component should render 1`] = `
       <SetupInstructionsLink />
     </EuiFlexItem>
   </EuiFlexGroup>
-  <EuiSpacer
```

Comment:
@chandlerprall see comments from @sqren and @nreese here: https://github.com/elastic/kibana/pull/31935#discussion_r260000513

Has the default size for this component changed?

Reply:
Default size remains the same, but has been moved from `defaultProps` to being applied at destructuring in the function component's signature, so React is not aware of the default.